bundles/org.openhab.binding.resol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="resol"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="vbuslan"/>
+		</supported-bridge-type-refs>
+
+		<label>Resol Device</label>
+		<description>Solar or system controller (or any other real device on the VBus) from Resol.</description>
+	</thing-type>
+
+	<thing-type id="emulatedEM">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="vbuslan"/>
+		</supported-bridge-type-refs>
+
+		<label>Emulated Extension Module (EM) Device</label>
+		<description>Emulation of an EM device which can be connected through the VBUS to resol controllers which support the EM devices.</description>
+	    <channels>
+            <channel id="relay1" typeId="relay"/>
+            <channel id="relay2" typeId="relay"/>
+            <channel id="relay3" typeId="relay"/>
+            <channel id="relay4" typeId="relay"/>
+            <channel id="relay5" typeId="relay"/>
+            
+            <channel id="temperature1" typeId="temperature"/>
+            <channel id="temperature2" typeId="temperature"/>
+            <channel id="temperature3" typeId="temperature"/>
+            <channel id="temperature4" typeId="temperature"/>
+            <channel id="temperature5" typeId="temperature"/>
+            <channel id="temperature6" typeId="temperature"/>
+
+            <channel id="resistor1" typeId="resistance"/>
+            <channel id="resistor2" typeId="resistance"/>
+            <channel id="resistor3" typeId="resistance"/>
+            <channel id="resistor4" typeId="resistance"/>
+            <channel id="resistor5" typeId="resistance"/>
+            <channel id="resistor6" typeId="resistance"/>
+        </channels>
+        
+        <config-description>
+            <parameter name="deviceId" type="integer" required="true" min="1" max="15">
+                <label>Module ID to be emulated</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions